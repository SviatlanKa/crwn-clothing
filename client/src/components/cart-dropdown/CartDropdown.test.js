import React from 'react';
import { shallow } from 'enzyme';
import CartDropdown from './CartDropdown';

it('expect render CartDropdown component', () => {
    const cartItems = [
        {
            id: 5,
            imageUrl: 'www.images/blue-hat.jpg',
            price: 100,
            name: 'Blue Hat',
            quantity: 3
        }
    ];

    const props = {
        cartItems: cartItems,
        history: "http://localhost:3000",
        dispatch: jest.fn()
    }
    const wrapper = shallow(<CartDropdown.WrappedComponent { ...props }/>);
    expect(wrapper).toMatchSnapshot();
});
