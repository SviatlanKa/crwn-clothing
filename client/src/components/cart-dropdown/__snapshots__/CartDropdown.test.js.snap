// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect render CartDropdown component 1`] = `
<div
  className="cart-dropdown"
>
  <div
    className="cart-items"
  >
    <Memo(CartItem)
      item={
        Object {
          "id": 5,
          "imageUrl": "www.images/blue-hat.jpg",
          "name": "Blue Hat",
          "price": 100,
          "quantity": 3,
        }
      }
      key="5"
    />
  </div>
  <CustomButton
    onClick={[Function]}
  >
    GO TO CHECKOUT
  </CustomButton>
</div>
`;
